administration/src/mui-modules/application/forms/StepSendForm.tsx

```diff
@@ -83,6 +91,12 @@ const StepSendForm: Form<State, Options, ValidatedInput, AdditionalProps> = {
           setState={setGivenInformationIsCorrectAndComplete}
           options={givenInformationIsCorrectAndCompleteOptions}
         />
+        <SubForms.hasAcceptedEmailUsage.Component
+          label='Meine E-Mailadresse darf im Sinne der Datenschutzerklärung für Kommunikationsmaßnahmen (z.B. Newsletter, Hinweise zu Gewinnspielen, ...) rund um das Thema Ehrenamt genutzt werden.'
```

Comment:
https://www.duden.de/rechtschreibung/E_Mail_Adresse
```suggestion
          label='Meine E-Mail-Adresse darf im Sinne der Datenschutzerklärung für Kommunikationsmaßnahmen (z.B. Newsletter, Hinweise zu Gewinnspielen, ...) rund um das Thema Ehrenamt genutzt werden.'
```